src/pages/products/[...id].tsx

```diff
@@ -0,0 +1,117 @@
+import { useState, useEffect } from 'react';
+import ControlBar from '@/lib/components/Core/ControlBar';
+import Logo from '@/lib/components/Core/Logo';
+import { Product } from '@/lib/types/product';
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { Button } from '@mantine/core';
+
+function Products() {
+  const [product, setProduct] = useState<Product>();
+  const [clickedReadMore, setClickedReadMore] = useState(false);
+  const [orderCount, setOrderCount] = useState(1);
+  const router = useRouter();
+  const { id } = router.query;
+  const MAX_CHARACTERS = 135;
+  let isMaxCharacters = false;
+  useEffect(() => {
+    console.log(id);
+    if (id) {
+      fetch(`https://fakestoreapi.com/products/${id}`)
+        .then((res) => res.json())
+        .then((data) => setProduct(data));
+    }
+  }, [id]);
+  function addOrderCount() {
+    setOrderCount((oldCount) => oldCount + 1);
+  }
+  function subractOrderCount() {
+    if (orderCount > 1) {
+      setOrderCount((oldCount) => oldCount - 1);
+    }
+  }
+  if (product?.description && product?.description.length > MAX_CHARACTERS) {
+    isMaxCharacters = true;
+  }
+  console.log(product);
+  return (
+    <>
+      <ControlBar />
+      <Logo />
+      {product && (
+        <div className="flex flex-col items-center justify-center">
+          <div className="text-yellow-600 bg-yellow-100 border border-yellow-200 rounded-xl p-4 text-center">
+            If you see this warning then, that means that the navbar isn&apos;t responsive yet or
```

Comment:
This warning is unnecessary because even though `Navbar` isn't responsive YET it doesn't mean that desktop users should also wait.